frontend/packages/dev-console/src/components/monitoring/dashboard/MonitoringDashboard.scss

```diff
@@ -1,7 +1,29 @@
-.odc-monitoring-dashboard__dropdown-options {
-    display: flex;
-    float: right;
+$no-wrap-breakpoint: 915px;
+.odc-monitoring-dashboard {
+  position: relative;
+  .odc-monitoring-dashboard__dropdown-options {
     align-self: flex-end;
-    margin-top: -65px;
-}
+    display: flex;
+    position: absolute;
+    right: 30px;
+    top: -65px;
+    @media(max-width: $no-wrap-breakpoint) {
+      display: block;
+      margin-left: 30px;
+      margin-top: 15px;
+      position: initial;
+    }
+    @media(max-width: $no-wrap-breakpoint) {
+      .monitoring-dashboards__dropdown-wrap {
+        display: inline-block;
+      }
+      .monitoring-dashboards__dropdown-title {
+        margin-right: 10px;
+      }
+    }
 
+    .monitoring-dashboards__dropdown-wrap:last-of-type {
+      margin-right: 0;
+    }
+  }
```

Comment:
You can merge these two, no? Don't recall if SASS plays well... but I think it would...